dashboard/test/models/script_test.rb

```diff
@@ -227,7 +227,7 @@ def populate_cache_and_disconnect_db
     scripts, _ = Script.setup([script_file_all_properties])
     script = scripts.first
 
-    assert_equal 21, script.properties.keys.length
+    assert_equal 20, script.properties.keys.length
```

Comment:
Should this be changing?

Reply:
Yea, I removed `hideable_stages` from the script_dsl parse output in this change also, forgot to call that out.

Reply:
No worries - Was just scanning for anything that jumped out. All looks good then!